app/src/main/java/com/zulip/android/activities/ZulipActivity.java

```diff
@@ -1042,10 +1042,20 @@ public void onClick(View view) {
                 displayChatBox(true);
                 displayFAB(false);
                 fabHidder.start();
+                showSoftKeyboard();
+                streamActv.requestFocus();
             }
         });
     }
 
+    /**
+     * Shows soft keyboard
+     */
+    private void showSoftKeyboard() {
+        InputMethodManager imm = (InputMethodManager) getSystemService(Context.INPUT_METHOD_SERVICE);
+        imm.toggleSoftInput(InputMethodManager.SHOW_FORCED,InputMethodManager.HIDE_IMPLICIT_ONLY);
```

Comment:
I think we won't need the second commit if we use [SHOW_IMPLICIT](https://developer.android.com/reference/android/view/inputmethod/InputMethodManager.html#SHOW_IMPLICIT) instead of SHOW_FORCED.